### DISCUSSION TITLE
Put the Build Phase setting of iOS when pod install.

### DISCUSSION THREAD

**Discussion by tihimsm** [invertase/react-native-firebase#4519] [Ideas]
I want to put the Build Phase setting of iOS when pod install.
https://firebase.google.com/docs/crashlytics/get-started?hl=ja#initialize-crashlytics

I think it's very good that this script is added automatically.
`$ {PODS_ROOT} / FirebaseCrashlytics / run`

But I also want to add Input Files.
`${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}`
`$(SRCROOT)/$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)`

**Comment by mikehardy** [comment]
@tihimsm we will happily merge PRs 🙏
